src/main/scala/org/broadinstitute/dsde/workbench/sam/service/CloudExtensions.scala

```diff
@@ -66,6 +68,8 @@ trait CloudExtensionsInitializer {
 trait NoExtensions extends CloudExtensions {
   override def isWorkbenchAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(true)
 
+  override def isSamSuperAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(true)
```

Comment:
Is there a todo to implement this function? Or if not, how does this work?

Reply:
`NoExtensions` is for running Sam without connecting to Google which we only do in tests. I'm not sure _why_ we have it defined here instead of as a test file or something like that, but that's why it's just stubbed out like this

Reply:
Yeah, I was suspecting it was for tests, but the fact that it wasn't in a spec folder had me second guessing myself. Sounds good.